OpenSim/Examples/ExampleMain/OutputReference/TugOfWar_Complete.cpp

```diff
@@ -201,11 +201,13 @@ int main()
 
         // Specify the paths for the two muscles
         // Path for muscle 1
-        muscle1->addNewPathPoint("muscle1-point1", ground, Vec3(0.0,0.05,-0.35));
-        muscle1->addNewPathPoint("muscle1-point2", *block, Vec3(0.0,0.0,-0.05));
+        auto& path1 = muscle1->updPath<GeometryPath>();
```

Comment:
A lot of this diff (e.g. the above 50-100 diff lines) demonstrate the problem with changing the API in a breaking way.

Because the `addNewPathPoint` was dropped (logically, cleaner), rather than `[[deprecated]]` (logically, unclean, but backwards-compatible), a large amount of changes are necessary throughout the codebase where the new code will fail in precisely the same way that the `[[deprecated]]`, runtime-checked backwards-compatible version would fail anyway.

Now imagine multiplying the number of changes necessary here by the number of lines of scripting code out in the wild (matlab scripts, whatever), guess what percentage of those developers are still around, what percentage of those will post issues, etc. - that's how much extra work is created vs. just supplying a basic backwards-compatible version.